great_expectations/experimental/column_descriptive_metrics/metrics.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import uuid
+from typing import Any, Sequence, Union
+
+import pydantic
+from pydantic import BaseModel, Field
+
+
+class MetricRepositoryBaseModel(BaseModel):
+    """Base class for all MetricRepository related models."""
+
+    class Config:
+        extra = pydantic.Extra.forbid
+
+
+class Value(MetricRepositoryBaseModel):
+    value: Any  # TODO: Better than Any
+
+
+class Metric(MetricRepositoryBaseModel):
```

Comment:
Let's stop the crime! Please introduce:
TableMetric
ColumnMetric
ColumnPairMetric
MultiColumnMetric
Metric is the base model (and is abstract)

Reply:
Added a note as well:
```
Note: This implementation does not currently take into account
    other domain modifiers, e.g. row_condition, condition_parser, ignore_row_if
```